dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -147,7 +310,7 @@
     {% if column.data_type|lower == 'boolean' %}
        {{ column.name }} boolean {%- if not loop.last -%},{%- endif -%}
     {% elif column.data_type|lower == 'character varying(256)' %}
-      {{ column.name }} varchar(255) {%- if not loop.last -%},{%- endif -%}
+      {{ column.name }} string {%- if not loop.last -%},{%- endif -%}
```

Comment:
Should this be string? I think it should still be varchar(255). What is the reason for this change?

Reply:
When I tried the original varchar(255) it errored and said that 'string' should be used instead.

<img width="929" alt="image" src="https://user-images.githubusercontent.com/30411048/225937033-75d6e532-00ae-41bb-872c-ea4623b62fb0.png">